addons/debugger/userscript.js

```diff
@@ -342,14 +343,44 @@ export default async function ({ addon, global, console, msg }) {
   });
   let logs = [];
   let scrollQueued = false;
-  const addItem = (content, thread, type) => {
-    const wrapper = document.createElement("div");
-    const span = (text, cl = "") => {
-      let s = document.createElement("span");
-      s.innerText = text;
-      s.className = cl;
-      return s;
-    };
+  class LogWrapper {
+    constructor() {
+      this.element = document.createElement("div");
+      this.element.className = "log";
+      return this.element;
+    }
+  }
+  class LogText {
+    constructor(text, className, useHtml = false) {
+      this.element = document.createElement("span");
+      this.element.className = className;
+      if (useHtml) {
+        this.element.innerHTML = text;
+      } else {
+        this.element.innerText = text;
+      }
+      return this.element;
+    }
+  }
+  class FeedbackWrapper {
+    constructor() {
+      const feedbackLog = new LogWrapper();
+      feedbackLog.append(
+        new LogText(
+          `${msg("feedback-log")} <a href="https://scratchaddons.com/feedback" target="_blank">${msg(
+            "feedback-log-link"
+          )}</a>`,
```

Comment:
safeMsg pls

Reply:
also, im not sure this works in every language... to separate a sentence

Reply:
```js
// maybe:
msg("feedback-log", { logLink: `<a href="https://scratchaddons.com/feedback" target="_blank">${msg("feedback-log-link")}</a>` })
```
```js
// for the locales
{
  ...
  "debugger/feedback-log": "We'd appreciate it if you'd share how you use this addon {logLink}",
  "debugger/feedback-log-link": "here"
}